extractor/src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInName/Example-All-In-One.java

```diff
@@ -0,0 +1,91 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <!-- AbbreviationAsWordInName Example 1 -->
+    <module name="AbbreviationAsWordInName">
```

Comment:
Not good, it is not how it is in checkstyle repo.
Please have a folder with few Example.java files, and all config are in separate files.
Just copy all examples AbbreviationAsWordInName and make their class to be empty as we don't care about class content and violations inside it

Reply:
I have pushed my changes trying best to understand on the review comment. But, how do i create a test case for all-in-one examples. I deleted the above file assuming we didn't want all the properties inside a Examples.java. So, how does the ConfigSerializer creeate an `extractor/src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInName/Configs/config-all-examples-in-one.xml`. This is the expected one.

Reply:
So, far the ConfigSerializer.java is working for all test cases.

Reply:
> Please have a folder with few Example.java files, and all config are in separate files.

have i addressed this?